explorer/utils/dockerHubUtils.ts

```diff
@@ -98,9 +98,19 @@ export interface ManifestHistoryV1Compatibility {
     created: string;
 }
 
+export interface ManifestTag {
```

Comment:
Do these actually exist this way in Docker Hub?

Reply:
We've got Docker Hub, ACR, private registries, and soon third-party registries (google, amazon, etc), so we can't make many assumptions.

Reply:
The response I was getting while debugging include this. That's why I added it.

Reply:
Was it Docker Hub or ACR?

Reply:
Extend would be better